### PR TITLE
Add nu pkgs

### PR DESCRIPTION
**Pull request by Yethal**
# Description

(description of your pull request here)

# Tests

Make sure you've done the following:

- [ ] Ensure your files have up to date metadata embedded inside
- [ ] (If submitting a module) Ensure all exported functions have documentation added to them
- [ ] (If submitting a module) Ensure all arguments to exported functions have documentation added to them
- [ ] (If submitting a module) Ensure all exported functions have examples added to them if usage is non obvious
- [ ] (If submitting a module) Ensure helper functions and variables are not exported from the module
- [ ] (If submitting a module) Ensure all exported functions have completion commands (unless non-feasible)
- [ ] (If submitting a standalone script) Ensure your script has a proper shebang at the top (`/usr/bin/env nu` should work in most circumstances)
- [ ] (If submitting a standalone script) Ensure your script has proper file permissions (Run `chmod +x` on the file before push)

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ jobs:
       - name: Setup Nu
         uses: hustcer/setup-nu@main
         with:
-          version: '0.63.0'
-      - name: Update json
+          version: '0.64.0'
```
Comment (skelly37) [inline .github/workflows/release.yml:23]: don't forget to make it up to date before merge

File: installer.nu [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        |lines
+        |append $"source ($nu-pkgs)"
+        |str collect (char nl)
+        |save $nu.config-path
+    } else {
+        print 'You will have to source the nu-pkgs file manually'
+    }
+} else {
+    print 'nu-pkgs already exists.'
+}
```
Comment (skelly37) [inline installer.nu:37]: We should raise a non-scoped errors here to make users aware that something went wrong.

Comment (Yethal) [reply installer.nu]: changed to error

File: modules/remoting.nu [hunk 3]
```diff
@@ -1,136 +1,72 @@
-#?name: remoting
+#?name: make
```
Comment (skelly37) [inline modules/remoting.nu:1]: Please get the newest remoting.nu and make.nu before merging

File: modules/nupac.nu [hunk 4]
```diff
@@ -34,6 +34,7 @@ def get-flag-value [
 # Path where packages will be installed, can be changed to any other directory provided it is added to NU_LIB_DIRS variable
 def scripts-path [] {
     $env
+    |get -i "NUPAC_DEFAULT_LIB_DIR"
     |get -i "NUPAC_DEFAULT_LIB_DIR"
```
Comment (skelly37) [inline modules/nupac.nu:38]: please remove one of them, something went wrong with the merge ig

File: testing/linter.nu [hunk 5]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env nu
+ls modules/
```
Comment (skelly37) [inline testing/linter.nu:2]: I think that all `*.nu` files should be lintered. The following pipeline should do until we have no dir inside a dir:

```suggestion
ls
|append (ls | where type == dir |get name| each {|it| ls $it})
|get name
```

File: modules/make.nu [hunk 6]
```diff
@@ -1,73 +1,411 @@
-#?name: make
-#?author: [Yethal,]
-#?version: 1.0.0
-#?short-desc: makefile completions for nu
+#?name: nupac
+#?author: [skelly37, Yethal]
+#?version: 0.1.0
+#?short-desc: package manager for nushell
```
Comment (skelly37) [inline modules/make.nu:4]: ?

File: installer.nu [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+        open $nu.config-path
+        |lines
+        |append $"source ($nu-pkgs)"
+        |str collect (char nl)
+        |save $nu.config-path
+    } else {
+        print 'You will have to source the nu-pkgs file manually'
+    }
+} else {
+    error make {
```
Comment (skelly37) [inline installer.nu:36]: We can use unspanned errors for better UX since 0.66

```suggestion
    error make -u {
```

Comment (skelly37) [reply installer.nu:36]: We could also add the `--unspanned` flag for the 3 `error make` ocurrences in the actual nupac.nu imo

Comment (Yethal) [reply installer.nu]: done

File: modules/make.nu (hunk 6 above)
Comment (Yethal) [reply modules/make.nu]: Fixed now

File: testing/linter.nu (hunk 5 above)
Comment (Yethal) [reply testing/linter.nu]: Linter now covers all .nu files (but modules are still linted separately from standalone scripts)

File: modules/nupac.nu (hunk 4 above)
Comment (Yethal) [reply modules/nupac.nu]: Done

File: modules/remoting.nu (hunk 3 above)
Comment (Yethal) [reply modules/remoting.nu]: Done

File: .github/workflows/release.yml (hunk 1 above)
Comment (Yethal) [reply .github/workflows/release.yml]: Done

File: .github/workflows/release.yml [hunk 8]
```diff
@@ -20,7 +20,7 @@ jobs:
       - name: Setup Nu
         uses: hustcer/setup-nu@main
         with:
-          version: '0.63.0'
-      - name: Update json
+          version: '0.65.0'
```
Comment (skelly37) [inline .github/workflows/release.yml:23]: Why not 0.66.2?

File: README.md [hunk 9]
```diff
@@ -8,7 +8,11 @@ Nu package manager written entirely in nu
 See [TODO.md](docs/TODO.md)
 
 ## Design Doc
-See [DESIGN.md](docs/DESIGN.md)
+See [DESIGN.md](DESIGN.md)
```
Comment (skelly37) [inline README.md:11]:
```suggestion
See [DESIGN.md](docs/DESIGN.md)
```

you broke it accidentally ig

File: modules/nupac.nu [hunk 10]
```diff
@@ -9,14 +9,15 @@
 #?url: https://github.com/skelly37/nupac/blob/main/modules/nupac.nu
 #?raw-url: https://raw.githubusercontent.com/skelly37/nupac/main/modules/nupac.nu
 #?keywords: [package, management]
+#?os: [windows, linux, macos, android]
```
Comment (skelly37) [inline modules/nupac.nu:12]:
```suggestion
#?os: [android, linux, macos, windows]
```

Let's keep it in alphabetical order

File: modules/remoting.nu [hunk 11]
```diff
@@ -10,6 +10,7 @@
 #?url: https://github.com/skelly37/nupac/blob/main/modules/remoting.nu
 #?raw-url: https://raw.githubusercontent.com/skelly37/nupac/main/modules/remoting.nu
 #?keywords: [ssh, remote, automation]
+#?os: [windows, linux, macos, android]
```
Comment (skelly37) [inline modules/remoting.nu:13]:
```suggestion
#?os: [android, linux, macos, windows]
```

same

**Review by skelly37 — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yml (hunk 8 above)
Comment (skelly37) [inline .github/workflows/release.yml:23]:
```suggestion
          version: '0.66.2'
```

Why not 0.66.2?

**Review by skelly37 — APPROVED** [review]

**Pull request merged**